src/clickableText.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Support for clickable text in instructions.
+ *
+ * When the text markdown contains:
+ *   [text](#clickable=id)
+ *
+ * This code will generate
+ *   <b data-id="id" class="clickable-text">text</b>
+ */
+
+module.exports = function clickableText() {
+  const Parser = this.Parser;
+  const tokenizers = Parser.prototype.inlineTokenizers;
+  const originalText = tokenizers.link;
```

Comment:
nit; this is a slightly misleading variable name, what about something like,

```suggestion
  const originalLinkTokenizer = tokenizers.link;
```

as in https://github.com/code-dot-org/remark-plugins/blob/d8d480504fdf36350fa8b802b7883c3a2b7b0718/src/link.js#L46

Reply:
Good call.  Done.